ynput/ayon-core
# Tray: Helper to make sure it is running

## Changelog Description
Implemented helper function `make_sure_tray_is_running` to start tray if is not running yet.

## Additional info
The function can be used to start tray if is not running, there might be caviads that have to be cleaned up over time, usually connected to environment variables, but that is hard to tackle without the issues.

Does not have real usage at this moment, but will be used with webactions.

## Testing notes:
1. Start tray.
2. Launch DCC (with python console e.g. maya, nuke etc.).
3. In DCC run this, which should not do anything:
```python
from ayon_core.tools.tray import make_sure_tray_is_running

make_sure_tray_is_running()
```
4. Exit tray.
5. Run the same code from 3.) which should start tray process.
